### DISCUSSION TITLE
ASIO Settings change by themselves

### DISCUSSION THREAD

**Discussion by JoeSouthern** [jamulussoftware/jamulus#1386] [Technical Support & Help] [answered]
I have version 3.7.  I'm new (:

I have my ASIO settings set to input Aux5 and Aux 6 on my Presonus Mixer and output to Channel 19 and Channel 20 on the same mixer. (I've tried various setups trying to fix this issue - this is my current one.).

It seems to work fine - so I roll along and then sometimes 2 minutes later or sometimes 18 minutes later (more on the faster side) and the settings change to my presonus Channel 1 and 2 for input and output.  I change them and then a little later it resets to the Channel 1 and 2 again. I'm listening or participating (slightly ha ha) to a jam going on with a server and then it all goes quiet. I check and sure enough my mixer settings reset. I set up a private server here and it does the same thing after just being connected for a bit ( a few minutes).

I've tried saving the mixer settings, but even with trying to load from that it doesn't appear to have saved my settings.

I use the Presonus in my daw and zoom, etc and haven't had any issues with the asio.

I switched to the ASIO4all and that works - settings hold for hours, but I've read where using the ASIO4all provides more latency than the manufacturers asio and I think I'm seeing that. I've checked for updates from presonus.

There was someone on the old forum having the exact same issue on his mac in November I think - I have Windows 10. It didn't appear that an answer was posted to his question.

Any ideas? I looked at the mixer save file and it doesn't appear to have anything in there about my asio settings that I could see.

**Comment by dcorson-ticino-com** [comment, marked as answer]
We have a new bug that has been discovered in version 3.7.0 which sounds similar https://github.com/jamulussoftware/jamulus/issues/1346
There is a test version mentioned there that you could try.

**Reply by JoeSouthern** [reply]
It sounds exactly like that. I did not see that post. I've been reviewing the code and was going to set up for a debug process to try to figure it out. I'll check out the test version. That's great - thanks a lot for your reply.

**Reply by JoeSouthern** [reply]
As an update - the test version fixed it or certainly appears to. I'll update if any issues are found later, but that fixed it.

**Comment by ann0see** [comment]
Could you please check the pre release version from the release tab? The bug fix was merged now

**Reply by JoeSouthern** [reply]
Ok - my steps -  Windows 10

Used -  Release 3.7.0devNightly1-cae8d2d (r3_7_0devNightly1)

Compiled it from the zipped source.

Took the Jamus.exe file and moved it to my music machine replacing the test debug  .exe I did a few days ago.

Did several connections including to local server - seems to be holding my input and output selections just fine. It's just been 10 minutes or so but I would have known if it wasn't working by now. Thanks!

**Reply by ann0see** [reply]
Good to hear!
